### ISSUE TITLE
Create RPM/SRPM for Fedora RedHat i386/amd64/arm 23 24 25 26

### ISSUE THREAD

**Issue by ckti** [cevap/ion#86]
Create packages for Fedora RedHat OS'es

**Comment by ckti** [comment]
Have fc25 x86_64 & SRPM in alpha state.

**Comment by ckti** [comment]
rpm and srpm are (currently) at [https://udic.000webhostapp.com/Ionomy/](https://udic.000webhostapp.com/Ionomy/) for Alpha Testing...still a bit to do, but getting there,

**Comment by ckti** [comment]
Site has changed to https://www.ioncoin.weebly.com currently uploading now

**Comment by ckti** [comment]
rpm site is now down.  1 or 2 more changes and this one is ready to put into fc 23-26 xwin & console

**Comment by ckti** [comment]
website is https://ioncoin.000webhostapp.com for the time being.  FC25 rpm & srpm are the only ones up atm

**Comment by ckti** [comment]
arm v7hl rpm up at https://ioncoin.000webhostapp.com/fed25/ioncoin-2.1.6.2-675ae7b.fc25.armv7hl.rpm

**Comment by ckti** [comment]
will be re-compiling with final version **bb68b3b** ty to aspa for reminding me about ssl compatability :)

**Comment by ckti** [comment]
Have Fedora arm 25  & 64 bit 26 version 2.1.6.3 now.. fedora environments are work in progress currently.

**Comment by ckti** [comment]
Getting armv7hl 23 building now...will start to transpose to 24,25,26 32/64 bit and add in splitting for the 3 packages ioncoin, iond, and ion-qt.

**Comment by cevap** [comment]
very good, just to mention, we have working ion-cli now too, ion-tx is not finished yet but could be already included too.

**Comment by ckti** [comment]
yes, noticed this on my last few builds.  had to adjust my start/stop scripts to use ion-cli :)

**Comment by ckti** [comment]
Getting armv7hl 23 building now...will start to transpose to 24,25,26 32/64 bit and add in splitting for the 3 packages ioncoin, iond, and ion-qt.

**Comment by ckti** [comment]
have arm src.rpms on version  cceacd6 up at https://ioncoin.000webhostapp.com still need to tease out the 3 packages from the srpm.

**Comment by ckti** [comment]
now getting the .specs right to build ioncoin for amd64 and armv7l. i386 will not be far behind.

**Comment by ckti** [comment]
ioncoin fedora 23, 24, 25, 26 i686, x86_64 and src.rpm up at https://ioncoin.000webhostapp.com fed 23 armv7hl is up as well. fed 24,25, & 26 armv7hl being compiled now.

**Comment by ckti** [comment]
ioncoin fedora 23, 24, 25, 26 i686, x86_64 and src.rpm up at https://ioncoin.000webhostapp.com fed 23 armv7hl is up as well. fed 24,25, & 26 armv7hl being compiled now.

**Comment by ckti** [comment]
Ioncoin fedora 23, 24, 25 & 26 all uploaded i686, x86_64 and arm7hl

**Comment by ckti** [comment]
have left building armhf packages for now, have the ioncoin rpms (i686,x86_64) and srpms for version 5bc80c4 up at https://ioncoin.000webhostapp.com.  will separate them into ioncoin iond and ion-qt.

**Comment by ckti** [comment]
have got a working split to ioncoin, iond,ion-qt and ion-docs now compiling for i686 & x86_64 fed 23,24,25,26 and starting into the armhf fed 23,24,25,26

**Comment by ckti** [comment]
updated and split packages are at https://ioncoin.000webhostapp.com/ fed 23, 24, 25, 26 RPMS and SRPMS

**Comment by ckti** [comment]
N.B.  These are beta installs

**Comment by ckti** [comment]
Have cleaned up the directories for both fedora and raspian.

**Comment by ckti** [comment]
Currently re-ordering the package contents
ion-2.1.6.3-5bc80c4.1.fc24.x86_64.rpm 
/usr/bin/ion-cli
/usr/bin/ion-tx
/usr/lib/libionconsensus.a
/usr/lib/libionconsensus.la
/usr/lib/pkgconfig
/usr/lib/pkgconfig/libionconsensus.pc
/usr/lib/systemd
/usr/lib/systemd/system
/usr/lib/systemd/system/iond.service
/usr/share/man/man1
/usr/share/man/man1/ion-cli.1
/usr/share/man/man1/ion-qt.1
/usr/share/man/man1/ion-tx.1
/usr/share/man/man1/iond.1

ion-doc-2.1.6.3-5bc80c4.1.fc24.x86_64.rpm 
(all docs)

ion-ioncoin-2.1.6.3-5bc80c4.1.fc24.x86_64.rpm 
/etc/sysconfig/iond
/usr/bin/ion-qt
/usr/bin/iond
/usr/share/pixmaps/ion.ico
/usr/share/pixmaps/ion128.png
/usr/share/pixmaps/ion128.xpm
/usr/share/pixmaps/ion16.png
/usr/share/pixmaps/ion16.xpm
/usr/share/pixmaps/ion256.png
/usr/share/pixmaps/ion256.xpm
/usr/share/pixmaps/ion32.png
/usr/share/pixmaps/ion32.xpm
/usr/share/pixmaps/ion64.png
/usr/share/pixmaps/ion64.xpm
/usr/share/pixmaps/nsis-header.bmp
/usr/share/pixmaps/nsis-wizard.bmp

ion-iond-2.1.6.3-5bc80c4.1.fc24.x86_64.rpm 
/etc/sysconfig/iond
/usr/bin/iond

ion-ion-qt-2.1.6.3-5bc80c4.1.fc24.x86_64.rpm 
/usr/bin/ion-qt
/usr/lib/pkgconfig
/usr/lib/pkgconfig/libionconsensus.pc
/usr/share/pixmaps/ion.ico
/usr/share/pixmaps/ion128.png
/usr/share/pixmaps/ion128.xpm
/usr/share/pixmaps/ion16.png
/usr/share/pixmaps/ion16.xpm
/usr/share/pixmaps/ion256.png
/usr/share/pixmaps/ion256.xpm
/usr/share/pixmaps/ion32.png
/usr/share/pixmaps/ion32.xpm
/usr/share/pixmaps/ion64.png
/usr/share/pixmaps/ion64.xpm
/usr/share/pixmaps/nsis-header.bmp
/usr/share/pixmaps/nsis-wizard.bmp

**Comment by ckti** [comment]
arm packages are now up-to-date at https://ioncoin.000webhostapp.com/

**Comment by ckti** [comment]
i686 x86_64 and arm builds are now up at https://ioncoin.000webhostapp.com/
NB_these are beta still

**Comment by ckti** [comment]
Working on pre-post installs for application menus and test to make sure that they work in supported spins.  Leaving ARM for the time being.

**Comment by ckti** [comment]
Currently getting packages to build on supported versions (and on fc23/fc24).  On revision 4 now.

**Comment by ckti** [comment]
have had to re-think the compilation.  verrry slow under vbox.   moving to a systemd-nspawn on a different system for compilations.

**Comment by ckti** [comment]
still not speedy, but much better.   Building fc25 now, fc23 and fc24 finished

**Comment by ckti** [comment]
fc23 and fc24 2.1.6.3 5bc80c4 available at https://ioncoin.000webhostapp.com

**Comment by ckti** [comment]
all 4 versions up at https://ioncoin.000webhostapp.com/ version 5bc80c4 plus md5sums for all packages
